README.md

```diff
@@ -1,35 +1,21 @@
-# ATM-3
-
-All The Mods 1.12
-======
-This is the official repository and issue-tracker for All The Mods 3 1.12
-  
-Does "All The Mods" *really* contain ALL THE MODS? No, of course not.
-  
-#### Curse Links: 
```

Comment:
What's the reason for removing the links to the other modpacks?

Reply:
The projects are part of an organization they can be found under the same creator. If you really feel like updating every readme advertising all of the other modpacks, would seem tedious. I feel like each pack should have it’s own tailored readme.